awx/ui_next/src/components/PaginatedDataList/PaginatedDataList.jsx

```diff
@@ -4,99 +4,54 @@ import { DataList } from '@patternfly/react-core';
 import { withI18n } from '@lingui/react';
 import { t } from '@lingui/macro';
 import { withRouter } from 'react-router-dom';
-import styled from 'styled-components';
 
-import ContentEmpty from '../ContentEmpty';
-import ContentError from '../ContentError';
-import ContentLoading from '../ContentLoading';
-import Pagination from '../Pagination';
-import DataListToolbar from '../DataListToolbar';
-import PaginatedDataListItem from './PaginatedDataListItem';
+import ListHeader from '@components/ListHeader';
+import ContentEmpty from '@components/ContentEmpty';
+import ContentError from '@components/ContentError';
+import ContentLoading from '@components/ContentLoading';
+import Pagination from '@components/Pagination';
+import DataListToolbar from '@components/DataListToolbar';
+
 import {
-  parseNamespacedQueryString,
```

Comment:
I like the changes here because they:
   - maintain the existing external contract between paginated data list and components that use it
   - simplify the _implementation_ of paginated data list by refactoring it into smaller, more testable components.

:+1:

Reply:
Agreed. This is a nice improvement!